tests/tracing/test_misc.py

```diff
@@ -1,67 +1,43 @@
 import pytest
-import gc
-import uuid
-import os
-from unittest import mock
 from unittest.mock import MagicMock
 
 import sentry_sdk
-from sentry_sdk import start_span, start_transaction, set_measurement
+from sentry_sdk import start_span, set_measurement, get_current_scope
 from sentry_sdk.consts import MATCH_ALL
-from sentry_sdk.tracing import Span, Transaction
 from sentry_sdk.tracing_utils import should_propagate_trace
 from sentry_sdk.utils import Dsn
 
 
-def test_span_trimming(sentry_init, capture_events):
-    sentry_init(traces_sample_rate=1.0, _experiments={"max_spans": 3})
```

Comment:
no longer supported, if we want to support `max_spans` we will need some logic in the span processor but it is not simple to support it